discovery_engine/example/lib/example.dart

```diff
@@ -28,8 +28,6 @@ Future<void> runExample() async {
     applicationDirectoryPath: appDirPath,
     feedMarkets: {const FeedMarket(langCode: 'de', countryCode: 'DE')},
     manifest: manifest,
-    headlinesProviderPath: '/newscatcher/v1/latest-headlines',
-    newsProviderPath: '/newscatcher/v1/search-news',
```

Comment:
❔ I don't see any information about removing this in the task or story descriptions, was it decided somewhere but not documented? Is there a plan to introduce other way for the app to configure the endpoints/providers? I think it would be nice to leave some form on control over this to the app to be able to do a/b test (that is if this is still our priority)

Reply:
i removed them as those paths moved into their own config struct. currently there is no way to set them from the outside, but we can easily arange for this via the `de_config` JSON config. should this be implemented now or only when needed later on @acrrd ?

Reply:
We still need to accept them from outside, but it should be optional. This should not use the `de_config`  but dart struct we have now.